### DISCUSSION TITLE
auto-complete uppercase commands - need a fix!

### DISCUSSION THREAD

**Discussion by cybersins** [ohmyzsh/ohmyzsh#10564] [Help]
Any idea why the ZSH might be doing this? I would like to fix it, or revert to my working ZSH.

OS: MacOS Monterey 12.1

  ![GIF on the ERROR](https://i.stack.imgur.com/ruDPC.gif)

**Comment by mcornella** [comment]
It seems that there's a command / alias named `Ls`, and zsh uses that instead of `ls`. You can debug the completion by pressing <kbd>CTRL+X</kbd> <kbd>?</kbd>. That's going to generate a trace of the completion, so you can debug what zsh is doing. You can upload that trace file here, if you can't make sense of it.
